src/services/dexes/uniswapV3/LiquidityService.ts

```diff
@@ -0,0 +1,41 @@
+import {Logger} from 'winston';
+import {inject, injectable} from 'inversify';
+import {Token} from '@uniswap/sdk-core';
+import {FeeAmount} from '@uniswap/v3-sdk';
+import JSBI from 'jsbi';
+
+import {BarChartTick} from './interfaces.js';
+import {ChainsIds} from '../../../types/ChainsIds.js';
+import {UniswapV3LiquidityCalculator} from './UniswapV3LiquidityCalculator.js';
+
+@injectable()
+export class LiquidityService {
```

Comment:
```suggestion
export class LiquiditySummingService {
```

if you can think about better name that reflect that this service is summing up liquidity, then feel free to change it, if not, please apply suggestion.

Reply:
I am terrible with names. This sounds a better name indeed. Done